ios/RPCModule.swift

```diff
@@ -262,22 +245,9 @@ class RPCModule: NSObject {
 
   func loadExistingWallet(server: String, chainhint: String) throws -> String {
```

Comment:
im not sure what happens in the case of load error

Reply:
This function is called from other with the same name which throw the error resolving a promise, this promise is intercepted by the RN layer which show an Alert to the user.